packages/browser/examples/assets/queries/buildingByIDQuery.rq

```diff
@@ -0,0 +1,71 @@
+# Common prefixes
```

Comment:
Common prefixes should be define one time.
Maybe create a common.rq file and at the time to make the request add it to the specific request content

Reply:
When loaded the query text add dynamically the common.rq text before ?

Reply:
See reply above
To make these queries lighter they have been simplified so unused prefixes in each query have been removed.